tag/reader.py

```diff
@@ -190,6 +188,23 @@ def _resolve_features(self):
             for child in self.featsbyparent[parentid]:
                 parent.add_child(child, rangecheck=self.strict)
 
+        # Replace top-level multi-feature reps with a pseudo-feature
+        for n, record in enumerate(self.records):
+            if not isinstance(record, Feature):
+                continue
+            if not record.is_multi:
+                continue
+            assert record.multi_rep == record
+            newrep = sorted(record.siblings + [record])[0]
+            if newrep != record:
+                for sib in sorted(record.siblings + [record]):
+                    sib.multi_rep = newrep
+                    if sib != newrep:
+                        newrep.add_sibling(sib)
+                record.siblings = None
+            parent = newrep.pseudoify()
+            self.records[n] = parent
```

Comment:
If the multifeature is unsorted or reverse sorted in the input, the sorting in the output may be wonky. Ensuring that the rep is the component with the lowest-coordinate interval was the only way I was able to get sane sorting behavior in feature graph traversal and output.